base_wamas_ubl/lib/wamas/wamas_grammar/art.py

```diff
@@ -51,7 +51,7 @@
             "type": "str",
             "length": 20,
             "dp": False,
-            "dict_key": False,
+            "dict_key": 'id',
```

Comment:
hi @nilshamerlinck could you please validate this mapping

Reply:
@QuocDuong1306 is checking example files/integration to confirm mapping